editor/test/selectors.js

```diff
@@ -839,4 +890,57 @@ describe( 'selectors', () => {
 			expect( isSavingNewPost( state ) ).to.be.false();
 		} );
 	} );
+
+	describe( 'getSuggestedPostFormat', () => {
+		it( 'returns null if cannot be determined', () => {
```

Comment:
Wording of these test names? Most of our other tests use `should`, and it seems good to be consistent... 

```js
it( 'should be painted yellow', ...
```